src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -82,24 +87,22 @@ public void before() {
 
   @After
   public void after() throws SQLException {
-    stopConnect();
     try (Connection c = pg.getEmbeddedPostgres().getPostgresDatabase().getConnection()) {
       try (Statement s = c.createStatement()) {
         s.execute("DROP TABLE IF EXISTS " + tableName);
       }
     }
     log.info("Dropped table");
+    stopConnect();
   }
 
   /**
    * Verifies that even when the connector encounters exceptions that would cause a connection
    * with an invalid transaction, the connector sends only the errant record to the error
    * reporter and establishes a valid transaction for subsequent correct records to be sent to
    * the actual database.
-   *
-   * @throws Exception
    */
-  @Test
+  @Test (expected = RuntimeException.class)
```

Comment:
Won't this cause the tests to pass if a `RuntimeException` gets thrown anywhere in the test, not just during the last call to `connect.kafka().consume()`?

Reply:
That's a good point. I think it would make more sense to just catch the Runtime Exception where we expect it to be thrown.

Reply:
Removed this with the new way of verifying that records have gone to the right place by using the waiting for offsets suggestion.